pkg/windows/windows.go

```diff
@@ -235,10 +232,13 @@ type Windows interface {
 	ConfigureWICD(string, *Authentication) error
 	// ConfigureKubeProxy ensures that the kube-proxy service is running
 	ConfigureKubeProxy() error
-	// EnsureRequiredServicesStopped ensures that all services that are needed to configure a VM are stopped
-	EnsureRequiredServicesStopped() error
-	// Deconfigure removes all files and services created as part of the configuration process
-	Deconfigure() error
+	// Cleanup runs the WICD cleanup command that ensures all services configured by WICD are stopped. If the bool is
+	// not set, this function will clean up the node object as well.
+	Cleanup(string, bool) error
```

Comment:
runWICDCleanup() is run  on the Windows instance, this method is also used in Configure() from nodeconfig to cleanup all services in case of error configuring the node.